src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj

```diff
@@ -24,6 +24,8 @@
     <NoDocumentationFile>true</NoDocumentationFile>
     <UseCommonOutputDirectory>true</UseCommonOutputDirectory>
     <ProjectSystemLayer>VisualStudio</ProjectSystemLayer>
+    <NeedsSigningToRelease>true</NeedsSigningToRelease>
+    <ProducingSignedVsix>true</ProducingSignedVsix>
```

Comment:
Also `NeedsSigningToRelease` is used only for the projects that produces binaries that will be signed and released. Since `VSL.Import.Targets` is imported by everybody we need a way to distinguish between projects that produce binaries and projects that are just helper projects which do not produce libraries